src/models/models.cpp

```diff
@@ -194,6 +194,7 @@ AutoModel::AutoModel(std::string modelPath, xft::DataType datatype) : Model() {
             case xft::DataType::fp16: setDecoder(new OptDecoder<float16_t>(modelPath)); break;
             case xft::DataType::bf16: setDecoder(new OptDecoder<bfloat16_t>(modelPath)); break;
             case xft::DataType::int8: setDecoder(new OptDecoder<int8_t>(modelPath)); break;
+            case xft::DataType::w8a8: setDecoder(new OptDecoder<w8a8_t>(modelPath)); break;
```

Comment:
need to add more config after case xft::DataType::bf16_int8:

case xft::DataType::bf16_w8a8:
    setDecoder(new HybridModel<OptDecoder, bfloat16_t, w8a8_t>(modelPath));
    break;

need to add more config after case xft::DataType::bf16_nf4:

case xft::DataType::w8a8_int8:
                setDecoder(new HybridModel<OptDecoder, w8a8_t, int8_t>(modelPath));
                break;
case xft::DataType::w8a8_int4:
                setDecoder(new HybridModel<OptDecoder, w8a8_t, uint4x2_t>(modelPath));
                break;
case xft::DataType::w8a8_nf4:
                setDecoder(new HybridModel<OptDecoder, w8a8_t, nf4x2_t>(modelPath));
                break;